input_ssml_text_longer.html

```diff
@@ -0,0 +1 @@
+
```

Comment:
```suggestion
The OpenMC data downloader offers command a line tool and a python API that both allow the downloading preprocessed h5 cross section files.
There is a pypi installer for the package and users can install the software with pip install openmc data downloader.

Once installed the command line tool can be used

All the options can be found using the minus minus help option

A library or libraries are always needed and these should be provided in order of preference.

As you can see there are several options available including JEFF 3.2, FENDL 3.1d, TENDL 2019, WMP and ENDF

The downloader will try to find isotopes in the preferentially specified by the order of the libraries provided.

Once the isotope is found it will be downloaded and named after the library and isotope and a cross sections xml file will also be made.

This create a reproducible nuclear data environment ideal.

To download specific isotopes from specific libraries like this.

-l for library and -i for isotope

running the same command will detect that the cross sections have been downloaded and skip the downloading

Elements can also be requested and they will be expanded to their natural abundances

-e for elements

Elements and isotopes can both be specified

the h5 files can be sent to a specific directory using the -d flag

h5 files needed for neutron and photon transport can be downloaded using the -p flag

The package can also be imported into a python script and used in conjunction with OpenMC to download the nuclear data required for the simulation.

In this example we see that OpenMC data download has been imported as odd .

an OpenMCmaterial has been made and that material has been passed to the just in time library generator with a preference for the fendl nuclear data library

this will now download the nuclear data and set the OpenMCcross section

the just in time download can also be called with a list of materials

and also an OpenMCmaterials object can be used

additionally the neutronics material maker Material objects and Multi Material objects can also be used.

The OpenMCcross section enviromental varible is automatically set to the new corss section.xml that has been downloaded and a simulation can be initialed using the recently downloaded cross sections.
```